NEWS

```diff
@@ -44,17 +44,34 @@ issues]].
   - Fixed a contraction for word "кое-где", now it is contracted
     according to rules.
   - Other minor fixes and improvements.
+Updated the Danish tables to the new 2022 Danish Braille standard
+  Thanks to Bue Vester-Andersen.
+  - Limited contraction level (grade 2l or 1.5)is no longer part of the Danish
+    Braille standard 2022.
+  - There is no longer any need for "literary" tables. In Danish Braille,
+    capital letters are now always indicated, and steps have been
+    taken in the standard to ensure better back-translation.
+  - As requested by The Danish Braille Committee, the tables for the
+    1993 standard will be kept as an option for at least a year.
+    All the 1993 tables have been renamed to include "_1993" in the
+    filename, so the new tables will be used as default	
 ** Other changes
 ** Deprecation notice
 - None
 ** Backwards incompatible changes
 ** Invisible changes
 ** New, renamed or removed tables
 *** New
-None
-
+- da-dk-g08.ctb da-dk-g16.ctb, da-dk-g18.ctb da-dk-g26.ctb, da-dk-g28.ctb
+  and da-dk-octobraille.dis now all refer to the corresponding tables
+  forDanish Braille 2022.
 *** Renamed
-None
+The old Danish tables for the now deprecated 1993 standard,
+  da-dk-g08.ctb, da-dk-g16.ctb, da-dk-g16-lit.ctb, da-dk-g18.ctb,
+  da-dk-g26.ctb, da-dk-g26-lit.ctb, da-dk-g26l.ctb, da-dk-g26l-lit.ctb,
+  da-dk-g28.ctb, da-dk-g28l.ctb and da-dk-octobraille.dis, have been
+  renamed to include "-1993" in the filename, i.e. da-dk-g08_1993.ctb,
+  da-dk-g16_1993.ctb etc.
```

Comment:
@egli Is this the right format?

Reply:
There is no standard to speak of. The whole thing is just to help downstream projects such as nvda. I think we usually use something like 
- old.ctb -> new.ctb